int_experian/int_experian_sfra/cartridge/static/default/js/edqUtils.js

```diff
@@ -278,7 +278,7 @@ countryDict.push({ key: "AF", value: "AFG" },
 	{ key: "ZW", value: "ZWE" });
 function countryAlpha3(incomingCountryIso2) { 
 	var iso2ToIso3CountryDict;
-	countryDict.forEach((val) => iso2ToIso3CountryDict = (incomingCountryIso2.match(val.key)) ? val.value : iso2ToIso3CountryDict);
+	countryDict.forEach(function(val) { iso2ToIso3CountryDict = (incomingCountryIso2.match(val.key)) ? val.value : iso2ToIso3CountryDict });
```

Comment:
What is the purpose of this change?

Reply:
Compatibility issues with IE11; the syntax "=>" when "foreach" was used was generating troubles to load the file on IE11; using a function syntax is compatible.